packages/secretlint/test/create-secretlintrc.test.ts

```diff
@@ -0,0 +1,67 @@
+// LICENSE : MIT
+"use strict";
+import assert from "assert";
+import path from "path";
+import fs from "fs";
+import os from "os";
+import { runConfigCreator } from "../src/create-secretlintrc";
+
+/**
+ *
+ * Create .secretlintrc.json
+ * - yarn test -g "create-secretlintrc-test"
+ *
+ * File Overview
+ *
+ * - tmpConfigDir
+ *   - A temporal directory with "secretlint-config"
+ *   - Create .secretlintrc.json on the directory
+ * - packageFilePath
+ *   - A path of package.json to use of this test
+ *   - When using this package.json, copy it to tmpConfigDir
+ * - expectedConfigFilePath
+ *   - A path of .secretlintrc.json to create by this test
+ */
+describe("create-secretlintrc-test", function () {
+    let tmpConfigDir: string;
+
+    beforeEach(function () {
+        tmpConfigDir = fs.mkdtempSync(path.join(os.tmpdir(), "secretlint-config"));
+        const packageFilePath = path.join(__dirname, "fixtures", "package.json");
+        fs.promises.copyFile(packageFilePath, tmpConfigDir + "/package.json");
+    });
```

Comment:
`fs.promises.copyFile` without `await`  is something wrong.

```suggestion
    beforeEach(async function () {
        tmpConfigDir = await fs.mkdtemp(path.join(os.tmpdir(), "secretlint-config"));
        const packageFilePath = path.join(__dirname, "fixtures", "package.json");
        await fs.copyFile(packageFilePath, path.join(tmpConfigDir + "package.json"));
    });
```

use await and promise APIs.

Reply:
Thanks for your comments!!
I did it
https://github.com/secretlint/secretlint/pull/306/commits/1efc702a48d294951f4f1c5902e1580427bcfa96